includes/components/class-admin.php

```diff
@@ -946,6 +949,261 @@ protected function get_extensions_tab() {
 		return $current_tab;
 	}
 
+	/**
+	 * Gets system report.
+	 *
+	 * @return array
+	 */
+	protected function get_report() {
+
+		global $wpdb;
+
+		// Get cURL version.
+		$curl_version = esc_html__( 'No', 'hivepress' );
+
+		if ( function_exists( 'curl_version' ) ) {
+			$curl_version = hp\get_array_value( curl_version(), 'version' ) . ', ' . hp\get_array_value( curl_version(), 'ssl_version' );
+		} elseif ( extension_loaded( 'curl' ) ) {
+			$curl_version = esc_html__( 'cURL installed but unable to retrieve version.', 'hivepress' );
+		}
+
+		// Get post types count.
+		$post_types = [];
+
+		foreach ( (array) $wpdb->get_results( "SELECT post_type AS 'type', count(1) AS 'count' FROM {$wpdb->posts} GROUP BY post_type;" ) as $post_type_count ) {
+			$post_types[] = [
+				'label' => sprintf( esc_html__( '%s:', 'hivepress' ), $post_type_count->type ),
+				'value' => $post_type_count->count,
+			];
+		}
+
+		// Get active plugins.
+		$active_plugins = array_filter(
+			array_map(
+				function ( $plugin ) {
+					$file = WP_PLUGIN_DIR . '/' . $plugin;
+
+					if ( file_exists( $file ) ) {
+						$headers = array_filter(
+							get_file_data(
+								$file,
+								[
+									'name'    => 'Plugin Name',
+									'version' => 'Version',
+									'url'     => 'Plugin URI',
+									'author'  => 'Author',
+								]
+							)
+						);
+
+						$value = [
+							'label' => sprintf( esc_html__( '%s:', 'hivepress' ), hp\get_array_value( $headers, 'name' ) ),
+							'value' => sprintf( esc_html__( 'by %1$s - %2$s', 'hivepress' ), hp\get_array_value( $headers, 'author' ), hp\get_array_value( $headers, 'version' ) ),
+							'url'   => hp\get_array_value( $headers, 'url' ),
+						];
+
+						return $value;
+					}
+				},
+				(array) get_option( 'active_plugins' )
+			)
+		);
+
+		// Get theme info.
+		$theme = wp_get_theme( get_template() );
+
+		$theme_info = [
+			[
+				'label' => esc_html__( 'Name:', 'hivepress' ),
+				'value' => $theme->get( 'Name' ),
+			],
+			[
+				'label' => esc_html__( 'Version:', 'hivepress' ),
+				'value' => $theme->get( 'Version' ),
+			],
+			[
+				'label' => esc_html__( 'Author URL:', 'hivepress' ),
+				'value' => $theme->get( 'AuthorURI' ),
+			],
+			[
+				'label' => esc_html__( 'Child theme:', 'hivepress' ),
```

Comment:
Avoid adding special characters in the beggining/end of custom strings, this would make them more re-usable (e.g. when used in different context not as a table column), add semicolons after the translation function